controller-api/src/main/java/com/yahoo/vespa/hosted/controller/api/role/Policy.java

```diff
@@ -142,7 +142,32 @@
     /** Access to /payment/notification */
     paymentProcessor(Privilege.grant(Action.create)
                                 .on(PathGroup.paymentProcessor)
-                                .in(SystemName.PublicCd));
+                                .in(SystemName.PublicCd)),
+
+    /** Read your own instrument information */
+    paymentInstrumentRead(Privilege.grant(Action.read)
+                                   .on(PathGroup.billingInstrument)
+                                   .in(SystemName.PublicCd)),
+
+    /** Ability to update tenant payment instrument */
+    paymentInstrumentUpdate(Privilege.grant(Action.update)
+                                     .on(PathGroup.billingInstrument)
+                                     .in(SystemName.PublicCd)),
+
+    /** Ability to remove your own payment instrument */
+    paymentInstrumentDelete(Privilege.grant(Action.delete)
+                                     .on(PathGroup.billingInstrument)
+                                     .in(SystemName.PublicCd)),
```

Comment:
What's the benefit of having these as three separate policies?

Reply:
They are assigned differently in roles?

Reply:
Alright